### ISSUE TITLE
Gravity Forms Not Working

### ISSUE THREAD

**Issue by scottcarlton** [Astoundify/wp-job-manager-contact-listing#6]
This plugin doesn't seem to function with gravity forms. Currently using gravity forms Version 1.7.12 "Job Submission" set as instructed in the video with the Listing ID and the no-reply@listingowner.com email set in notifications. Several things are not functioning. One the Listing ID <a href="http://wordpress.org/support/topic/gravity-forms-entry_id-not-working?replies=1">reported issue on wp</a> and two the email notification . No email is sent either to the user or the posting "Application email/URL:" set email address. The email is recorded in the gravity forms entries section. This occurs even if Listing ID is not set. So submitting the form works, but there is no confirmation on the front-end or via the set emails. Has anyone else had this issue? If so is there any help out there?

**Comment by KZeni** [comment]
I know this topic is closed, but I found that using `{entry_id}` with it dynamically populated had it leaving "{entry_id}" in Gravity Form's submission data while the email notification then had it referencing the Gravity Forms submission ID rather than the job listing ID (alternative setups seems to just leave it as "{entry_id}" in all cases).

The fix I found? I switched to using `{embed_post:ID}` for the Listing ID field's default value (doesn't need to be dynamically populated). This has it reporting the correct listing ID in the email and Gravity Forms submission data. It's possible that the documentation for this could/should be updated with this method.

Hopefully this helps anyone that came across this same issue.
